docs/system/container-metrics.md

```diff
@@ -0,0 +1,108 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Container
+--->
+
+# Semantic Conventions for Container Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+## Container Metrics
+
+### Metric: `container.cpu.utilization`
+
+This metric is optional.
+
+<!-- semconv metric.container.cpu.time(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `container.cpu.time` | Counter | `ns` | Total CPU time consumed [1] |
```

Comment:
Why `ns`? OTel recommendation is to use the basic measures if possible. We use `s` for `system.cpu.time`. I think it should the same for the `container.cpu.time`

Reply:
The metric will be reported as a double type. I don't think we lose any meaningful precision. I'd prefer to be consistent with OTel recommendations. Kubelet stats receiver also uses seconds https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/kubeletstatsreceiver/metadata.yaml#L103

Reply:
updated: [1ae7244](https://github.com/open-telemetry/semantic-conventions/pull/282/commits/1ae72441e2ae5ed196598336fc9e22826f457096)